modules/RTC/TPCUtils.js

```diff
@@ -1,25 +1,26 @@
 import { getLogger } from '@jitsi/logger';
 import transform from 'sdp-transform';
 
+import CodecMimeType from '../../service/RTC/CodecMimeType';
 import { MediaDirection } from '../../service/RTC/MediaDirection';
 import { MediaType } from '../../service/RTC/MediaType';
 import { getSourceIndexFromSourceName } from '../../service/RTC/SignalingLayer';
+import { STANDARD_CODEC_SETTINGS } from '../../service/RTC/StandardVideoSettings';
+import VideoEncoderScalabilityMode from '../../service/RTC/VideoEncoderScalabilityMode';
 import { VideoType } from '../../service/RTC/VideoType';
 import browser from '../browser';
 import FeatureFlags from '../flags/FeatureFlags';
 
 const logger = getLogger(__filename);
 const DESKTOP_SHARE_RATE = 500000;
-const LD_BITRATE = 200000;
-const SD_BITRATE = 700000;
 const SIM_LAYER_1_RID = '1';
 const SIM_LAYER_2_RID = '2';
 const SIM_LAYER_3_RID = '3';
+const VIDEO_CODECS = [ CodecMimeType.AV1, CodecMimeType.H264, CodecMimeType.VP8, CodecMimeType.VP9 ];
 
-export const HD_BITRATE = 2500000;
-export const HD_SCALE_FACTOR = 1;
-export const LD_SCALE_FACTOR = 4;
-export const SD_SCALE_FACTOR = 2;
+export const HD_SCALE_FACTOR = 1.0;
+export const LD_SCALE_FACTOR = 4.0;
+export const SD_SCALE_FACTOR = 2.0;
```

Comment:
Doesn't it make sense to adjust these values when One enables 4K? Maybe it should be part of the config? I can see us wanting to use a scale factor of 6 in a Spot room configured for 4K for example.

Reply:
Yes, but I would think we should address it in a different PR since its not related to this feature.

Reply:
Makes sense!